vmware/ngc/NGC-AdapterManager/src/main/java/org/opensds/vmware/ngc/adapters/oceanstor/Constants.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package org.opensds.vmware.ngc.adapters.oceanstor;
```

Comment:
My understanding is that it now supports LVM as well.  Why is this an oceanstor specific package?

Reply:
There is storage adapter. So any third party storage like oceanstor can also implement that storage interface to support their backend storage.